### ISSUE TITLE
[Bug] Trim Errors

### ISSUE THREAD

**Issue by SaltSouls** [BeansGalaxy/Beans-Backpacks-2#120]
## Description:
When loading the backpack mod, I get a ton of these errors for various mods.
```
[11Apr2024 11:32:04.983] [Worker-ResourceReload-18/ERROR] [net.minecraft.client.renderer.texture.atlas.sources.PalettedPermutations/]: unable to apply palette to minecraft:trims/backpacks/sentry_pigsteel
java.lang.IllegalArgumentException: null
```
I would imagine this is because the mod is trying to register these trims/create the palettes for them without checking to see if said mods are even installed. While this doesn't cause a crash, it does spam the console log, and make it kinda difficult to discern other problems.

## Log Files:
[latest.log](https://gist.github.com/SaltSouls/4e40bbbc4c412760399e8f30dff7d375) | [debug.log](https://gist.github.com/SaltSouls/1e1528e9c286d8cd8e4bd62c0f861dfb)

**Comment by BeansGalaxy** [comment]
This is because I added support for the trim materials from Pigsteel & Enderite in the last updates but without them installed, minecraft can't find those entries and throws errors. Didn't realize the log was getting spammed until after I posted this version so I'll have to setup a way to register trims that can check if their mod's installed. 

Since it's not a critical issue and just log spam, I'll fix it in the next update instead of rolling it back and posting the fix now.
